README.md

```diff
@@ -1 +1,230 @@
-# ursa-rfcs
\ No newline at end of file
+# Ursa RFCs
+[Ursa RFCs]: #ursa-rfcs
+
+Many changes, including bug fixes and documentation improvements can be
+implemented and reviewed via the normal GitHub pull request workflow.
+
+Some changes though are "substantial", and we ask that these be put through a
+bit of a design process and produce a consensus among the Ursa community.
+
+The "RFC" (request for comments) process is intended to provide a consistent
+and controlled path for major changes to enter Ursa Core and other official
+project components, so that all stakeholders can be confident about the
+direction Ursa is evolving in.
+
+This process is intended to be substantially similar to the Rust RFCs process,
+customized as necessary for use with Ursa. The README.md and
+0000-template.md were initially forked from [Rust
+RFCs](https://github.com/rust-lang/rfcs).
+
+
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+  - [Opening](#ursa-rfcs)
+  - [Table of Contents]
+  - [When you need to follow this process]
+  - [Before creating an RFC]
+  - [What the process is]
+  - [The RFC life-cycle]
+  - [Reviewing RFCs]
+  - [Implementing an RFC]
+  - [Help this is all too informal!]
+  - [License]
+
+
+## When you need to follow this process
+[When you need to follow this process]: #when-you-need-to-follow-this-process
+
+You need to follow this process if you intend to make "substantial" changes to
+Ursa or any of its sub-components including but not limited to Ursa
+Core, Ursa Supply Chain, Ursa Seth, the various Ursa SDKs, or the
```

Comment:
What are these exactly? Did I miss something?

Reply:
oops. cut and paste error. i'll push up a fixup.